### PR TITLE
Issue 408 - Consistent top level headers

### PR DESCRIPTION
**Pull request by oemel09**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes https://github.com/open-horizon/open-horizon.github.io/issues/408
- Consistent top level headers
- Consistent indentation
- copyright updates

## Type of change

Please delete options that are not relevant.

- [x] This change requires a documentation update

## How Has This Been Tested?

View these pages at:
- https://open-horizon.github.io/docs/anax/docs/attributes/
- https://open-horizon.github.io/docs/anax/docs/api/
- ~https://open-horizon.github.io/docs/anax/docs/agent_container_manual_deploy/~
- https://open-horizon.github.io/docs/anax/docs/managed_workloads/

## Checklist:

- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: docs/agent_container_manual_deploy.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+lastupdated: "2023-02-09"
 title: "Agent in a Container"
 description: Instructions for starting an agent in a container on Linux
 
 parent: Agent (anax)
 nav_order: 2
 ---
 
-## Starting an agent in a container on Linux
+# Starting an agent in a container on Linux
```
Comment (johnwalicki) [inline docs/agent_container_manual_deploy.md:12]: There is a larger set of suggested edits to the agent_container_manual_deploy.md 
See #3636 
We should do the edits of this file in that issue.

File: docs/agent_container_manual_deploy.md [hunk 2]
```diff
@@ -47,17 +47,17 @@ If the management hub you are using uses Secure Socket Layer (SSL) encryption, t
 2. Prior to starting the container, export the following variables:
 
    * HZN_AGENT_IMAGE=\<name of the agent container image in the repo\>
-     * This should include the url for the repository if it is not the default one.
+      * This should include the url for the repository if it is not the default one.
```
Comment (johnwalicki) [inline docs/agent_container_manual_deploy.md:50]: The indentation of the sub-bullets is correct.   The alignment should be directly under the first character of parent bullet.  See the correct rendering at https://open-horizon.github.io/docs/anax/docs/agent_container_manual_deploy/

Same comment for the below edits.  The doc is already correct.

**Review by johnwalicki — COMMENTED** [review]
The rest of the edits look good.

File: docs/agent_container_manual_deploy.md (hunk 1 above)
Comment (oemel09) [reply docs/agent_container_manual_deploy.md]: Okay, I removed this file from the PR

**Review by johnwalicki — APPROVED** [review]
Thanks for the doc improvements.

**Pull request merged**